.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,5 +12,8 @@ Fixes #
 
 #### Changes proposed in this pull request:
 
--
--
+ <!-- Describe how this PR will make our project even more awesome.
+ If this solves an issue, please reference it in this PR. -->
```

Comment:
the _Fixes #_ thing at the top is to reference Issues, so I don't think it's necessary here as well.
I see people are making few PRs without opening an issue, should we mention in template that your PR will be closed without merge if we do not see issue tagged with it. @divyamrastogi ?

Reply:
> If this solves an issue
is not the case, because a PR should close at least 1 issue from the issue board.

Reply:
@ritikrishu  Please tell me the steps required (changes) from my side to close this.

Reply:
hey @optimistanoop, sorry I missed your comment here somehow. So the comment in PR template that says _"If this solves an issue, please reference it in this PR"_, I'd like to mention, _"Please provide reference to active issue here"_ (at the top where it says Fixes #my_issue_number). Also a warning, _"An active issue is required for a PR to be reviewed. If you do not provide one, you pull request might be closed by collaborators"_

Reply:
@ritikrishu @ParthS007  Review changes are done. Please have a look and close this.